src/applications/hca/tests/e2e/utils/navigation.js

```diff
@@ -122,34 +58,6 @@ export const advanceFromHouseholdToSubmit = (testData, props = {}) => {
   cy.location('pathname').should('include', '/confirmation');
 };
 
-export const advanceToHealthInsurance = testData => {
```

Comment:
Since we're starting in-progress, we don't need to advance to this section any longer.